src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -502,14 +502,17 @@ namespace OpenMS
       writeOutFeaturesAndChroms_(chromatograms, featureFile, out_featureFile, store_features, chromConsumer);
     }
 
-    std::vector<int> prm_map;
+    // (iii) map transitions to individual DIA windows for cases where this is
+    // non-trivial (e.g. when there is m/z overlap and a transition could be
+    // extracted from more than one window
+    std::vector<int> tr_win_map; // maps transition k to dia map i from which it should be extracted
     if (prm_)
     {
       // Here we deal with overlapping PRM / DIA windows: we only want to extract
       // each peptide from a single window and we assume that PRM windows are
       // centered around the target peptide. We therefore select for each peptide
       // the best-matching PRM / DIA window:
```

Comment:
comment how best-matching is defined?

Reply:
"Best matching" is slightly different for PRM or PASEF,  for PASEF is described on lines 545-546.

Reply:
Would it make sense to expose this information more prominent to the user?
E.g., 
https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/nightly/html/classOpenMS_1_1OpenSwathWorkflow.html#details

If it is just an unimportant implementation detail it is of course not needed, but I imagined someone might ask "How do you deal with overlapping m/z windows" etc.

Reply:
Good idea! I've added a description (see  https://github.com/OpenMS/OpenMS/pull/5911/commits/61089c972d08eefa6da52fd8d5e27acda46bd79c)

I'm not sure if the description is too verbose so I've left this conversation open.